cmd/gvproxy/main.go

```diff
@@ -112,11 +111,7 @@ func main() {
 		},
 		DNS: []types.Zone{
 			{
-				Name:      "apps-crc.testing.",
-				DefaultIP: net.ParseIP("192.168.127.2"),
-			},
-			{
-				Name: "crc.testing.",
+				Name: "containers.internal.",
```

Comment:
Can you keep `crc.testing.host`, it is used by [podman-machine-cni](https://github.com/containers/podman-machine-cni/blob/main/plugins/meta/podman-machine/config.go#L12) at the moment. I like `containers.internal` and we should change it in podman-machine-cni but for now I would like to keep backwards compatibility. We cannot guarantee that gvproxy on the host and podman-machine-cni in the VM are updated at the same time.

Reply:
Oh yes very good point! I add it back.

Reply:
done thx